scout/server/blueprints/variant/templates/variant/clinvar.html

```diff
@@ -262,7 +262,7 @@ <h5>
                       <td style="width:30%">Condition HPO term(s):<br>(associated to the variant)</td>
                       <td>
                         {% for hpo in pheno_hpos %}
-                          <input type="checkbox" id="conditions@{{pinned._id}}" value="HPO_{{hpo}}" name="conditions@{{pinned._id}}"><a href="http://hpo.jax.org/app/browse/term/{{hpo}}"
+                          <input type="checkbox" id="hpo_terms@{{pinned._id}}" value="{{hpo}}" name="hpo_terms@{{pinned._id}}"><a href="http://hpo.jax.org/app/browse/term/{{hpo}}"
```

Comment:
To simplify the downstream parsing I've modified the original form fields to distinguish between OMIM and HPO terms.